### ISSUE TITLE
Add slash commands to dpy

### ISSUE THREAD

**Issue by Sas2k** [Rapptz/discord.py#7799]
### Summary

a way to use slash  commands in dpy

### What is the feature request for?

discord.ext.commands

### The Problem

well since discord is going to make the *api* change by removing prefixes and etc... I think it would be good to have slash commands

### The Ideal Solution

the ideal solution is I hope would be another decorator you can put below the `@commands.command` (in cog, you know what I mean)
something like this
```python
@commands.command(name="slash")
@commands.slash()
def slash():
    await ctx.send("the actual thing doesn't have to be like this but I think it should be something like this but similar")
```

### The Current Solution

well for now the current solution is to use [discord-py-slash-command](https://pypi.org/project/discord-py-slash-command/)
library

### Additional Context

Thanks for reviving the library.
and making this library in the first place

**Comment by drobotk** [comment]
App commands will be available in version 2.0.0 of the library, which is already in public testing and you can use it by installing from the master branch

**Comment by dannynotsmart** [comment]
the latest version of the master branch of discord.py does support slash commands!
check this gist out: https://gist.github.com/Rapptz/c4324f17a80c94776832430007ad40e6
